mobile/lib/features/brag/brag.dart

```diff
@@ -0,0 +1,426 @@
+                          begin: Alignment.bottomCenter,
+                          end: Alignment.topCenter,
+                          colors: [
+                            gradient1,
+                            gradient2,
+                          ],
+                        ),
+                      )),
+                    ),
+                    Expanded(
+                      flex: 2,
+                      child: Container(
+                          decoration: BoxDecoration(
+                        gradient: LinearGradient(
+                          begin: Alignment.bottomCenter,
+                          end: Alignment.topCenter,
+                          colors: [
+                            gradientColor0,
+                            gradient1,
+                          ],
+                        ),
+                      )),
+                    ),
+                    Expanded(
+                        flex: 2,
+                        child: Container(
+                          color: tenTenOnePurple,
+                        )),
+                  ],
+                ),
+                Align(
+                    alignment: Alignment.topLeft,
+                    child: SizedBox(
+                      width: 55,
+                      height: 55,
+                      child: Padding(
+                          padding: const EdgeInsets.all(8.0),
+                          child: Container(
+                            decoration: BoxDecoration(
+                                color: Colors.white,
+                                border: Border.all(
+                                  color: tenTenOnePurple,
+                                ),
+                                borderRadius: const BorderRadius.all(Radius.circular(5))),
+                            child: Padding(
+                                padding: const EdgeInsets.all(2.0),
+                                child: SvgPicture.asset(
+                                  'assets/10101_logo.svg',
+                                )),
+                          )),
+                    )),
+                Align(
+                  alignment: Alignment.topRight,
+                  child: SizedBox(
+                      width: 55,
+                      height: 55,
+                      child: Padding(
+                        padding: const EdgeInsets.all(8.0),
+                        child: Container(
+                            decoration: BoxDecoration(
+                                color: Colors.white,
+                                border: Border.all(
+                                  color: tenTenOnePurple,
+                                ),
+                                borderRadius: const BorderRadius.all(Radius.circular(5))),
+                            child: Padding(
+                              padding: const EdgeInsets.all(2.0),
+                              child: SvgPicture.asset('assets/10101-qr-transparent.svg',
```

Comment:
🙃 I think the meme is a little bit too obscured (looking at the video). It would be ideal if the meme could be a bit more visible with its original colours.

I would particularly avoid covering the text with our overlay. If we remove some of the elements I suggested I think this would be possible.

Reply:
> I would particularly avoid covering the text with our overlay. If we remove some of the elements I suggested I think this would be possible.

This needs to be considered when creating the meme images. I've updated them which looks better now.